inc/template-tags.php

```diff
@@ -400,7 +400,7 @@ function loose_customize_css() {
 		$custom_css .= '.menu-logo {float:left;}';
 		$custom_css .= '.navbar-navigation ul, .nav-social {display:block;}';
 		$custom_css .= '.loose-featured-slider, .loose-featured-slider .featured-image, .loose-featured-slider .no-featured-image {height:' . absint( get_theme_mod( 'home_page_slider_height', 500 ) ) . 'px;}';
-		$custom_css .= '.entry-content a {color:' . esc_attr( get_theme_mod( 'content_link_color', '#000' ) ) . ';}';
+		// $custom_css .= '.entry-content a {color:' . esc_attr( get_theme_mod( 'content_link_color', '#000' ) ) . ';}';
```

Comment:
@jayvin05 not sure why this needs to be commented out. This is the color users may set up in Customizer. So if anyone set up this already now it will stop working, right? May we do so in some backward compatible way?